test/e2e/QuerySettings.testa.js

```diff
@@ -0,0 +1,85 @@
+import assert from 'assert';
+import { startApp, stopApp } from './helpers/hooks';
+import { selectKeyspace, loadKeyspace, cleanKeyspace, waitForQueryCompletion, clearInput } from './helpers/actions';
+import { waitUntil } from './helpers/utils';
+
+jest.setTimeout(100000);
+
+describe('Query Settings', () => {
+  let app;
+
+  beforeAll(() => {
+    loadKeyspace('gene');
+  });
+
+  beforeEach(async () => {
+    app = await startApp();
+    const isAppVisible = await app.browserWindow.isVisible();
+    assert.equal(isAppVisible, true);
+    await selectKeyspace('gene', app);
+
+    // open settings
+    await app.client.click('.settings-tab');
+    assert.equal(await app.client.isExisting('.settings-tab-content'), true);
+
+    // ensure Loading Roleplayers setting is enabled
+    const isLoadingRoleplayersEnabled = await app.client.isSelected('.load-roleplayers-switch input');
+    if (!isLoadingRoleplayersEnabled) {
```

Comment:
This is the check I was referring to earlier.
This is not necessarily wrong, but let's try not to add too many things in the `beforeEach`, for example if it's only one test fiddling with the role-players-switch maybe we can put back the switch to the original position in the test itself, or maybe we can clean up a few things in the `afterEach`, I guess this is just a general warning not to concentrate too many things in the beforeEach. 
Can we maybe just start the app and have a `resetOriginalSettings()` function? It should be enough to delete the json file that stores the preferences to have a clean fresh app that is not affected by previous tests